canonical/ubuntu-pro-client
# chore: rename repo GPG keys from -advantage to -pro  (SC-758)

## Why is this needed?
This PR solves all of our problems because we will have no more keys called 'trusty', and our keys are the same for all releases - `esm-infra` was the first and last one.
Besides that, advantage goes away and pro is there now.

oh and it
Fixes: #1539

## Test Steps
- attach on older version
- update
- see keys changed names
- see everything works

## Checklist

## Does this PR require extra reviews?
 - [x] Yes - touching debian/ scripts
